SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Utils/Interest/ResultTypes/ResultTypesTest.cpp

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Tests/TestDefinitions.h"
+#include "Mocks/SpatialClassInfoManagerMock.h"
+
+#include "Utils/Interest/ResultTypes.h"
+
+#define RESULT_TYPES_TEST(TestName) \
+	GDK_TEST(Core, ResultTypes, TestName)
+
+// Sanity check tests for creating result types.
+// Run tests inside the SpatialGDK namespace in order to test static functions.
+namespace SpatialGDK
```

Comment:
AFAIK, we don't usually use `SpatialGDK` namespace in test files. It would probably suit better to have the constant and utility function definitions be in anonymous namespace. And the test functions have no namespace at all.

Reply:
good shout, have done this, turns out free functions is what was required to make this nicer